docs/oid-info.md

```diff
@@ -58,11 +95,90 @@ the git ref that the workflow run was based upon.
 This specifies the username identity in the OtherName Subject Alternative Name, as
 defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6).
 
+
+### 1.3.6.1.4.1.57264.1.8 | Source Repository URI
+
+Should include a fully qualified source repository URL that the build was based
```

Comment:
This one says "should" but we don't include should in other extensions. Is this referring to that the URL SHOULD be fully qualified, but may not be? Suggestion to rephrase: "Source repository URL that the build was based on. SHOULD be fully qualified. For example: blah", and then do the same with other URL extensions.

If being fully qualified is a MUST, we should note this since we'll need to enforce that.

Reply:
> Is this referring to that the URL SHOULD be fully qualified, but may not be?

👍 good spot, updated all the examples to make this clearer.